extensions/gdx-jnigen/src/com/badlogic/gdx/jnigen/parsing/JavaMethodParser.java

```diff
@@ -145,31 +158,49 @@ public String getName () {
 		public String toString () {
 			return "Argument [type=" + type + ", name=" + name + "]";
 		}
+
+		/** This only checks whether, the ArgumentType is the same, the name is ignored! 

```

Comment:
This is not really ideal.
It's needed for checking whether a list of arguments is equal.
I could have created a method that compares a list of arguments instead of relying on `List#equals()`, but I'm not sure whether a method for this special kind of comparison (Only comparing the `ArgumentType`) would be better.